docs/NextJs/Next.js API routes/API middlewares.md

```diff
@@ -0,0 +1,8 @@
+---
+id: API MiddleWares
```

Comment:
Invalid ID format. Since we use IDs for routing or URLs, please format the ID like this: `id: api-middlewares`